manifests/plugin/battery.pp

```diff
@@ -0,0 +1,17 @@
+# https://collectd.org/wiki/index.php/Plugin:Uptime
+class collectd::plugin::battery (
+  $ensure   = 'present',
```

Comment:
please add a datatype for `$ensure` as well.

Reply:
done